utils/gyb_syntax_support/ExprNodes.py

```diff
@@ -462,7 +462,7 @@
          traits=['Parenthesized'],
          children=[
              Child('Backslash', kind='BackslashToken'),
-             Child('Delimiter', kind='RawStringDelimiterToken', 
```

Comment:
Unrelated change?

Reply:
I changed my editor to avoid trimming whitespace in C++ files to not break formatting in the Swift repo, but forgot to do the same for Python files. Reverted.